packages/flutter/lib/src/material/text_field.dart

```diff
@@ -151,6 +151,10 @@ class TextField extends StatefulWidget {
     this.textAlign = TextAlign.start,
     this.textDirection,
     this.readOnly = false,
+    bool copy,
```

Comment:
These already have a lot of options... would it maybe make sense to give them a single toolbarOptions property that encapsulates all of these and nicely groups them?